pkg/workload/histogram/exporter/util.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2024 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package exporter
+
+import (
+	"regexp"
+	"time"
+
+	"github.com/codahale/hdrhistogram"
+	"github.com/gogo/protobuf/proto"
+	prom "github.com/prometheus/client_model/go"
+)
+
+func sanitizeOpenmetricsLabels(input string) string {
+	regex := regexp.MustCompile(`[^a-zA-Z0-9_]`)
```

Comment:
Nit: for improved readability, I'd swap the order and rename as `invalidFirstCharRegex` and `invalidCharRegex`. 

Could also lift these into globals to avoid recompilation, but this function isn't on a critical path, so that's effectively a premature optimization :)

Reply:
On second thought, it does appear on a critical path because it's invoked from `ConvertHdrHistogramToPrometheusMetricFamily`.